lib/i18n.php

```diff
@@ -38,17 +38,3 @@ function gutenberg_get_jed_locale_data( $domain ) {
 
```

Comment:
I might expect we should be able to do away with this file altogether? Do we need `gutenberg_get_jed_locale_data` ? It's still used in `client-assets.php`, but I would assume we should be able to rely on the core behavior now to bootstrap the localized data.